management/management_request.go

```diff
@@ -285,3 +285,13 @@ func Stringify(v interface{}) string {
 	}
 	return string(b)
 }
+
+// Sort configures a request to sort data by the selected field.
+// Use 1 to sort ascending and -1 to sort descending.
+func Sort(sort string) RequestOption {
```

Comment:
Should we use a string parameter for the field, and another boolean for the sort direction? Instead of expecting the exact string that the underlying API endpoint expects.

Reply:
I'd noodled around with this a bit, initially I had `Sort("name", "-1")` then I looked at Java/.Net and they had a style like what I went with (`Sort("name:-1")`) but I'm on the fence about it as they all kinda hide what the real meaning of 1/-1 are so on a scan through usage it's not immediately obvious what's being performed.

I did consider doing `SortAscending("name")` and `SortDescending("name")` to make it clearer but figured two might be overkill.

Reply:
Agree that "1" and "-1" don't really express much about what they mean.

>I did consider doing SortAscending("name") and SortDescending("name") to make it clearer

How are we already handling similar cases in the public API, if any?
BTW these seem quite readable to me.

Reply:
There isn't really anything too like this unfortunately, I guess the closest might the the `Query` API allowing passing [similar strings](https://github.com/auth0/go-auth0/blob/main/management/management_request.go#L241-L244)